lib/dev-server.js

```diff
@@ -0,0 +1,52 @@
+const path = require('path');
+const webpack = require('webpack');
+const WebpackDevServer = require('webpack-dev-server');
+
+const getConfig = require('./client-config.js');
+
+console.info('Running development webpack server...');
+
+module.exports = (rootPath, entry, destination, port = 3000, proxyPort = 3001) => {
+  const mode = 'development';
+  const compiler = webpack(getConfig({ name: 'dev-client', version: '0' }, rootPath, mode, entry, destination));
+  const options = {
+    publicPath: `http://localhost:${port}/app/`,
+    disableHostCheck: true,
+    hot: true,
+    inline: true,
+    historyApiFallback: true,
+    proxy: [
+      {
+        context: () => true,
+        target: {
+          port: proxyPort
+        }
+      }
+    ],
+
+    quiet: false,
+    noInfo: true,
+    watchOptions: {
+      aggregateTimeout: 300,
+      poll: 1000
+    },
+
+    stats: { colors: true },
+    headers: {
+      'Access-Control-Allow-Origin': '*'
+    }
+  };
+
+  new WebpackDevServer(compiler, options)
```

Comment:
Why do we need a dev server in the build step?

Reply:
It isn't a build step. The only reason why i added dev-server here is that we already have client webpack config and dependencies in `extensions-cli` (and dev-server options are the same for all extensions).